index.js

```diff
@@ -6,31 +6,40 @@
  *
  * A simple boolean input type
  */
-const validation = require('./lib/validation.js');
+const clone = require('lodash/clonedeep');
+const inpugCheckbox = require('input-plugin-checkbox');
 
+const utils = require('./lib/utils');
+
+// clone plugins for each input
+const boolean = clone(inpugCheckbox);
+
+// make replacements in html
+boolean.html = boolean.html.replace(/checkbox\./g, 'boolean.');
```

Comment:
Not going to want to do this. `input type="checkbox"` will be replaced with `input type="boolean"` and that's not what we want